themes/bootstrap3/js/lightbox.js

```diff
@@ -168,6 +168,14 @@ VuFind.register('lightbox', function Lightbox() {
       }
       obj.url += parts.length < 2 ? '' : '#' + parts[1];
     }
+    if (window.lightboxParent){
```

Comment:
The prevalent style has a space after the closing bracket, I'd like to follow it.

Reply:
fixed